From 3924d1936ab4b8b75458844ec1b86a5608a07f6e Mon Sep 17 00:00:00 2001
From: Magnus Damm <damm@opensource.se>
Date: Mon, 17 Feb 2014 16:19:45 +0900
Subject: ARM: shmobile: Remove Lager DT reference legacy clock bits

Lager DT reference is these days built for multiplatform
only which means that CCF comes with the package. Remove
unused legacy code ifdefs to clean up the code.

Signed-off-by: Magnus Damm <damm@opensource.se>
Acked-by: Laurent Pinchart <laurent.pinchart@ideasonboard.com>
Signed-off-by: Simon Horman <horms+renesas@verge.net.au>
(cherry picked from commit 6197c08ab81cf1430cbe7346fd6ca0b5a18580bf)
Signed-off-by: Simon Horman <horms+renesas@verge.net.au>
---
 arch/arm/mach-shmobile/board-lager-reference.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/arch/arm/mach-shmobile/board-lager-reference.c b/arch/arm/mach-shmobile/board-lager-reference.c
index 6ba8b3b41609..440aac36d693 100644
--- a/arch/arm/mach-shmobile/board-lager-reference.c
+++ b/arch/arm/mach-shmobile/board-lager-reference.c
@@ -88,7 +88,6 @@ static void __init lager_add_du_device(void)
 
 static void __init lager_add_standard_devices(void)
 {
-#ifdef CONFIG_COMMON_CLK
 	/*
 	 * This is a really crude hack to provide clkdev support to platform
 	 * devices until they get moved to DT.
@@ -126,9 +125,6 @@ static void __init lager_add_standard_devices(void)
 			clk_put(clk);
 		}
 	}
-#else
-	r8a7790_clock_init();
-#endif
 
 	r8a7790_add_dt_devices();
 	of_platform_populate(NULL, of_default_bus_match_table, NULL, NULL);
-- 
2.1.2

